modules/nosql/manager_redis.go

```diff
@@ -59,8 +60,84 @@ func (m *Manager) GetRedisClient(connection string) redis.UniversalClient {
 		name: []string{connection, uri.String()},
 	}
 
+	opts := getRedisOptions(uri)
+	tlsConfig := getRedisTLSOptions(uri)
+
+	clientName := uri.Query()["clientname"]
+
+	if nil != clientName && 0 < len(clientName) {
```

Comment:
Do not write if statements in reverse format.

```suggestion
	if clientName != nil && len(clientName) > 0 {
```

Reply:
> reverse format.

Yoda format: https://eslint.org/docs/rules/yoda

Reply:
Ah, sorry, habit! Swapped around 👍🏾